lib/Reader/abi.cpp

```diff
@@ -29,9 +29,9 @@
 using namespace llvm;
 
 // Static class with helpers for the Microsoft x86-64 ABI.
-class X86_64_Win64 {
+class XArchWin64 {
```

Comment:
The naming convention checks are generally not enforced in LLVM's formatting checks. So, violations are not checked/flagged there.